mandrill_to_keenio/recipe.json

```diff
@@ -0,0 +1,33 @@
+{
+    "recipes":[
+        {
+            "title": "Mandrill events to Keen.io collection",
+            "description": "Mandrill events to Keen.io collection",
+            "data": {
```

Comment:
just to make a note, that data on step2 will be filled by user. it can be deleted if you think it is necessury